packages/template-retail-react-app/app/static/translations/compiled/en-XA.json

```diff
@@ -1969,6 +1969,20 @@
       "value": "]"
     }
   ],
+  "credit_card_fields.tool_tip.security_code_aria_label": [
```

Comment:
![image](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/92a40ff2-2d5d-47aa-b933-e0ce0cb92638)
Missing translation for en-GB